### DISCUSSION TITLE
How to show passed classes, methods in HTML Report

### DISCUSSION THREAD

**Discussion by mufasalg0** [xunit/xunit#2600] [Question] [answered]
When I run my xUnit tests with command **_dotnet test --logger html_** , test report does not show the classes/method that passed. Is there any way to add to test report the ones that passed?

Thank you!!

**Comment by bradwilson** [comment, marked as answer]
Duplicate of #2596.

In short: we don't control the output from `dotnet test`, the VSTest team does. You can get TRX logs from `dotnet test`, which you can then transform either with your own hand-written XSL-T or some other (possibly third party) report generator.
